perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -400,10 +405,15 @@ bool BicycleTracker::measureWithShape(
 {
   // if the input shape is convex type, convert it to bbox type
   autoware_auto_perception_msgs::msg::DetectedObject bbox_object;
-  if (object.shape.type != autoware_auto_perception_msgs::msg::Shape::BOUNDING_BOX) {
-    utils::convertConvexHullToBoundingBox(object, bbox_object);
-  } else {
+  if (object.shape.type == autoware_auto_perception_msgs::msg::Shape::BOUNDING_BOX) {
     bbox_object = object;
+  } else if (object.shape.type == autoware_auto_perception_msgs::msg::Shape::CYLINDER) {
+    // convert cylinder to bbox
+    bbox_object.shape.dimensions.x = object.shape.dimensions.x;
```

Comment:
[imo]
This may become too large for long vehicles, use `sqrt(x**2 + y**2)` may better.

Reply:
Oh, I noticed this is invalid comment. Please ignore it :bow

Reply:
It will become a short-fat square bounding box :)